doc/api/next_api_changes/behavior/23299-TAC.rst

```diff
@@ -0,0 +1,6 @@
+mpl.rc_context no longer resets the value of :rc:`backend`
```

Comment:
The rest are fine, but this`:rc:` will end up far too long in the title.

Reply:
done!